js/generate.js

```diff
@@ -38,23 +38,24 @@ function fillCell(el, data, image) {
 }
 
 function importSVG(svgurl, el, postHook) {
-    var xhr = new XMLHttpRequest();
-    xhr.open("GET",svgurl);
-    xhr.responseType = "document";
-    xhr.onload = function (e) {
-	if (e.target.status == "200" || e.target.status == "304") {
-	    var svg = e.target.response.documentElement;
-	    svg.querySelector("style").remove();
-	    el.appendChild(svg);
-	    if (postHook) {
-		postHook(el);
-	    }
-	}
-    };
-    xhr.send();
+    fetch(svgurl, {mode:'cors'}).then(response => response.text())
+        .then(function(svg) {
+            if (svg) {
+                var d = document.createElement("p");
+                d.innerHTML = svg;
+                d.querySelector("style").remove();
+                el.appendChild(d);
+                if (postHook) {
+                    postHook(el);
+                }
+            }
+        }).catch(function(e) {
+            console.error(e);
```

Comment:
here you use function(), but below you use a console.xxx.bind(). Would be great if the two bound functions could just be declared a variable:

```JS
const consoleErr = console.error.bind(console);
...
.catch(consoleErr);
```

Or some such...